lib/constants.js

```diff
@@ -1,4 +1,15 @@
 module.exports = {
   ROOT_GITHUB: 'https://github.com/zapier/zapier-platform-schema',
-  DOCS_PATH: 'docs/build/schema.md'
+  DOCS_PATH: 'docs/build/schema.md',
+  MUTUALLY_EXCLUSIVE_FIELDS: [
+    ['children', 'list'], // This is actually a Feature Request (https://github.com/zapier/zapier-platform-cli/issues/115)
+    ['children', 'dict'], // dict is ignored
+    ['children', 'type'], // type is ignored
+    ['children', 'placeholder'], // placeholder is ignored
+    ['children', 'helpText'], // helpText is ignored
+    ['children', 'default'], // default is ignored
+    ['dict', 'list'], // Use only one or the other
+    ['dynamic', 'dict'], // dict is ignored
+    ['dynamic', 'choices'] // choices are ignored
+  ]
```

Comment:
These exclusive fields are only for `FieldSchema`, correct? Putting them here in `constants.js` is a bit confusing. Without looking at the other modules, it's not easy to tell what it is for only by the name of `MUTUALLY_EXCLUSIVE_FIELDS`. I suggest keeping `incompatibleFields` in `mutuallyExclusiveFields.js` and import it in `FieldSchema.js`. If that creates a circular import, maybe we can do the opposite, i.e., initialize `incompatibleFields` in `FieldSchema.js` and import it in `mutuallyExclusiveFields.js`. Does that make sense?